website/templates/profile/oauth_app_detail.mako

```diff
@@ -0,0 +1,110 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Application Detail</%def>
+<%def name="content()">
+<style type="text/css">
+    .border-box {
```

Comment:
Actually, since this is "non-critical" page-specific CSS, this can be put in its own CSS file, i.e. `project-settings-applications-detail.css`, which is loaded with `require()` in `project-settings.applications-detail.js`.